src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -249,13 +257,19 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
             case PredT(args) =>
               if (n.args.isEmpty && args.isEmpty) noMessages
               else multiAssignableTo.errors(n.args map exprType, args)(n) ++ n.args.flatMap(isExpr(_).out)
-            case c => Violation.violation(s"This case should be unreachable, but got $c")
+            case c => Violation.violation(s"This caseA should be unreachable, but got $c")
```

Comment:
you rewrote "case" multiple times to "caseA", "caseB", etc. Did you do this for debugging? If so, I would revert these changes. (For debugging, we can always use the line number reported in the stack trace)

Reply:
Fixed